lib/utils.js

```diff
@@ -68,6 +68,14 @@ function shuffle(array) {
     return (array);
 }
 
+function splitSubstr(str, len) {
```

Comment:
Did you end up not using this function?

Reply:
No, that's not needed any longer. I'll remove it.